osf-packager/src/main/java/org/dataconservancy/cos/packaging/OsfContentProvider.java

```diff
@@ -337,44 +236,17 @@ private static FileInfo contentFromUrl(String filename, String contentUrl, Relat
     }
 
     /**
-     * Creates a temporary directory under {@code java.io.tmpdir}.  The directory will be uniquely named, so as to
-     * avoid any filename conflicts with simultaneously executing downloads (e.g. another packager running in another
-     * JVM).
-     *
-     * @return the temporary directory
-     * @throws IOException if the directory cannot be allocated
-     */
-    static File allocateTempDir() throws IOException {
```

Comment:
It took me a bit to understand what happened here, but I understand and am not using the temporary directory code as originally intended.